Python/pywarpx/picmi.py

```diff
@@ -445,26 +445,43 @@ def initialize_inputs(self):
 
 class ElectromagneticSolver(picmistandard.PICMI_ElectromagneticSolver):
     def init(self, kw):
-        assert self.method is None or self.method in ['Yee', 'CKC'], Exception("Only 'Yee' and 'CKC' FDTD are supported")
+        assert self.method is None or self.method in ['Yee', 'CKC', 'PSATD'], Exception("Only 'Yee', 'CKC' and 'PSATD' are supported")
 
         self.do_pml = kw.pop('warpx_do_pml', None)
         self.pml_ncell = kw.pop('warpx_pml_ncell', None)
+        self.v_galilean = kw.pop('warpx_v_galilean', None)
```

Comment:
Here some other psatd options that could be added.
```suggestion
        if self.method == 'PSATD':
            self.periodic_single_box_fft = kw.pop('warpx_periodic_single_box_fft', None)
            self.fftw_plan_measure = kw.pop('warpx_fftw_plan_measure', None)
            self.do_current_correction = kw.pop('warpx_do_current_correction', None)
```